rest/route/github.go

```diff
@@ -30,9 +31,10 @@ const (
 	githubActionReopened    = "reopened"
 
 	// pull request comments
-	retryComment   = "evergreen retry"
-	patchComment   = "evergreen patch"
-	triggerComment = "evergreen merge"
+	retryComment         = "evergreen retry"
+	refreshStatusComment = "evergreen status"
```

Comment:
I have a slight preference for "evergreen refresh". 
I think it works nicely with "evergreen retry". One will take the tests and retry them, and on will simply refresh them.

Reply:
alright refresh it is!